silo-oracles/deploy/chainlink-v3-oracle/configs/sonic.json

```
@@ -56,5 +56,15 @@
         "secondaryHeartbeat": 0,
         "normalizationDivider": 100000000,
         "normalizationMultiplier": 0
+    },
+    "scUSD_USDC_USD": {
```

I'm getting a feeling, we should use prefixes for configs eg 
```suggestion
    "CHAINLINK_scUSD_USDC_USD": {
```

it might get messy soon if we will not